lib/util/is-builtin.js

```diff
@@ -0,0 +1,17 @@
+"use strict"
+
+const {
+    NodeBuiltinModules,
+} = require("../unsupported-features/node-builtins.js")
+
+const builtins = new Set(Object.keys(NodeBuiltinModules))
+
+/**
+ * @param {string} name The name of the module
+ * @returns {boolean}
+ */
+function isBuiltin(name) {
+    return builtins.has(name)
+}
```

Comment:
I originally tried using [is-builtin-module@3.2.1](https://www.npmjs.com/package/is-builtin-module/v/3.2.1) but it was missing `node:test`.

Thus I decided that, as we have the data already, we may as well use it :)